packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -1,5 +1,13 @@
 <template>
```

Comment:
Because the content area is very narrow on some screens the empty states get very squeezed, probably increasing the breakpoint for grid-cols should fix this
<img width="1039" alt="Screenshot 2024-12-18 at 17 00 33" src="https://github.com/user-attachments/assets/789f4caa-ab10-453b-ab7d-3d625c19eb77" />

Reply:
I was going to do it in another branch, but there is a new design for the empty state. I've just done it in this branch now:
<img width="885" alt="image" src="https://github.com/user-attachments/assets/7a816b77-e30a-40d6-a663-77f952d16235" />